contracts/oracle.sol

```diff
@@ -21,35 +21,27 @@
 pragma solidity ^0.6.11;
 
 import "./externals/base64.sol";
-import "./externals/provableAPI_0.6.sol";
+import "./externals/ECDSA.sol";
 import "./externals/SafeMath.sol";
 import "./externals/strings.sol";
 import "./internals/controllable.sol";
 import "./internals/date.sol";
 import "./internals/ensResolvable.sol";
 import "./internals/parseIntScientific.sol";
 import "./internals/tokenWhitelistable.sol";
-import "./internals/transferrable.sol";
 
 /// @title Oracle provides asset exchange rates and conversion functionality.
-contract Oracle is ENSResolvable, usingProvable, Transferrable, Base64, Date, Controllable, ParseIntScientific, TokenWhitelistable {
```

Comment:
Why did we chose to ditch the "transferrable" and ultimately ditch the Claim method ?